tests/e2e/e2e_test.go

```diff
@@ -116,6 +122,237 @@ var _ = ginkgo.AfterSuite(func() {
 })
 
 var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
+	ginkgo.It("can create blockhains", func() {
+		existingSubnetID := ""
+		ginkgo.By("start with blockchain specs", func() {
+			color.Outf("{{green}}sending 'start' with the valid binary path:{{/}} %q\n", execPath1)
+			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+			resp, err := cli.Start(ctx, execPath2,
+				client.WithBlockchainSpecs([]*rpcpb.BlockchainSpec{
+					{
+						VmName:  "subnetevm",
+						Genesis: "tests/e2e/subnet-evm-genesis.json",
+					},
+				}),
+			)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+			color.Outf("{{green}}successfully started:{{/}} %+v\n", resp.ClusterInfo.NodeNames)
+		})
+
+		ginkgo.By("wait for network to be healthy", func() {
+			color.Outf("{{blue}}wait for health{{/}\n")
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+			_, err := cli.Health(ctx)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+		})
+
+		ginkgo.By("get status", func() {
+			color.Outf("{{blue}}get status for existing subnet ID{{/}}\n")
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+			subnetDeployed := false
+		LOOP:
```

Comment:
could you avoid using labels here? suggest using the wait on "subnet creation" test. probably we can put that wait on a helper here
and get the subnet id in another step
I mean, split this into two ginkgo.By, one for waiting CustomChainsHealthy, by using some helper, and other
for obtaining the subnet id